DataFormats/L1TGlobal/interface/GlobalAlgBlk.h

```diff
@@ -92,12 +90,6 @@ class GlobalAlgBlk
 
 private:
 
-    /// orbit number
-    int m_orbitNr;
-
-    /// bunch cross number of the actual bx
-    int m_bxNr;
-
```

Comment:
why were these removed from the uGT data format ?
what is an alternative method to monitor the orbit and bx in which the various L1 triggers are firing ?
do you know if the same information is available somewhere else, e.g. from the TCDS data ?